src/filelock/_api.py

```diff
@@ -106,13 +106,13 @@ def is_locked(self) -> bool:
         """
         return self._lock_file_fd is not None
 
-    def acquire(self, timeout: Optional[float] = None, poll_intervall: float = 0.05) -> AcquireReturnProxy:
+    def acquire(self, timeout: Optional[float] = None, poll_interval: float = 0.05, **kwargs) -> AcquireReturnProxy:
```

Comment:
Please don't accept everything, the only thing we do need to accept is the old variant.

Reply:
At lines 151-152, it will raise `TypeError` when the user provides something other than the old variant.

Reply:
Yeah, I saw. But we don't need that, just express the intent via the interface, not fixup code.